packages/core/src/components/components.md

```diff
@@ -22,8 +22,10 @@
 @page overlay
 @page popover
 @page portal
-@page progress
+@page progress-bar
 @page sliders
```

Comment:
move sliders after spinner (minor alph break but it keeps the progress components together)

Reply:
Maybe we can address this in a follow up, cause we have other problems too (tag inputs far from inputs for instance)?

Reply:
no man, it's just moving one line around in a file you've already edited. please address here.